src/abstractdataframe/abstractdataframe.jl

```diff
@@ -592,14 +594,17 @@ end
 completecases(df::AbstractDataFrame, cols::AbstractVector) =
     completecases(df[cols])
 
+# TODO: update docstrings after deprecation of keepeltype
 """
-    dropmissing(df::AbstractDataFrame)
-    dropmissing(df::AbstractDataFrame, cols::AbstractVector)
-    dropmissing(df::AbstractDataFrame, cols::Union{Integer, Symbol})
+    dropmissing(df::AbstractDataFrame; keepeltype::Bool=true)
```

Comment:
I'd call the argument `disallowmissing` for consistency with the function with the same name.